File: src/main/java/com/redbox/RedboxApplication.java
package com.redbox;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.context.annotation.PropertySource;
import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
import org.springframework.scheduling.annotation.EnableScheduling;

@SpringBootApplication
@PropertySource("classpath:.env")
public class RedboxApplication {

    public static void main(String[] args) {
        SpringApplication.run(RedboxApplication.class, args);
    }

}

File: migration/src/main/kotlin/com/redbox/RedboxApplication.kt
package com.redbox

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.context.annotation.PropertySource

@SpringBootApplication
@PropertySource("classpath:.env")
class RedboxApplication

fun main(args: Array<String>) {
    runApplication<RedboxApplication>(*args)
}
